react-components/src/components/RevealContainer/RevealResourcesContext.tsx

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type CogniteModel } from '@cognite/reveal';
+import { type ReactElement, createContext, useContext, useState, type ReactNode } from 'react';
+
+type RevealResourcesContextType = {
```

Comment:
I think we don't have a convention of adding `Type` to types we are using, so I suggest to rename it to `RevealResourcesContext`.